appsec/tests/extension/headers_collection_14.phpt

```diff
@@ -0,0 +1,133 @@
+--TEST--
+All headers are collected when track_user_signup_event_automated is triggered by user
+--INI--
+extension=ddtrace.so
+datadog.appsec.log_file=/tmp/php_appsec_test.log
+datadog.appsec.log_level=debug
+datadog.appsec.enabled=1
+--ENV--
+HTTP_X_FORWARDED_FOR=7.7.7.7
+DD_TRACE_CLIENT_IP_HEADER_DISABLED=true
+HTTP_X_FORWARDED_FOR=7.7.7.6,10.0.0.1
```

Comment:
There's no point keep testing that the same headers are collected over and over. This just makes the tests difficult to read and ensures that when we modify the set of headers of we collect we need to change 20+ tests minimum.